### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jplatte**
Some things I noticed while working on the new Login API.

### CODE REVIEW

File: crates/matrix-sdk-crypto-nodejs/src/requests.rs [hunk 1]
```diff
@@ -56,7 +56,7 @@ pub struct KeysQueryRequest {
 
     /// A JSON-encoded object of form:
     ///
-    /// ```
+    /// ```json
```
Comment (poljar) [inline crates/matrix-sdk-crypto-nodejs/src/requests.rs:59]: Right, hit that one as well but forgot to push the patch I have somewhere.

**Review by poljar — APPROVED** [review]

File: crates/matrix-sdk-crypto-nodejs/src/requests.rs (hunk 1 above)
Comment (Hywan) [reply crates/matrix-sdk-crypto-nodejs/src/requests.rs:59]: Oops 🙊

**Review by Hywan — APPROVED** [review]
Thanks!

**Comment by jplatte** [comment]
No idea what's up with codecov, will merge manually I guess.

**Pull request merged**